cmd/crowdsec-cli/bouncers.go

```diff
@@ -130,20 +128,20 @@ To list/add/delete bouncers
 			}
 		},
 	}
-	cmdBouncersAdd.Flags().StringVarP(&keyName, "name", "n", "", "name to assigned for the api key")
 	cmdBouncersAdd.Flags().StringVarP(&keyIP, "ip", "i", "", "ip address of the blocker")
 	cmdBouncersAdd.Flags().IntVarP(&keyLength, "length", "l", 16, "length of the api key")
 	cmdBouncers.AddCommand(cmdBouncersAdd)
 
 	var cmdBouncersDelete = &cobra.Command{
 		Use:     "delete",
 		Short:   "delete bouncer",
-		Long:    `delete bouncer`,
-		Example: `cscli bouncers delete --name test [--ip 1.2.3.4]`,
-		Args:    cobra.MaximumNArgs(1),
+		Long:    `delete bouncer with the given name`,
+		Example: `cscli bouncers delete MyTestBouncer`,
```

Comment:
small fix, but can you use the same name for all bouncer in example ? upper you used  `MyBouncerName`
maybe you can just write : `<bouncer_name>` ?